mne_bids/utils.py

```diff
@@ -60,6 +60,20 @@ def _mkdir_p(path, overwrite=False, verbose=False):
             raise
 
 
+def _parse_bids_filename(fname):
```

Comment:
you need a test ;-)

Reply:
typically private functions are not tested explicitly. But these lines should be covered since they are being called by the `write_raw_bids` function